core/src/main/kotlin/org/evomaster/core/problem/rest/OpenApiAccess.kt

```diff
@@ -62,16 +62,32 @@ object OpenApiAccess {
     }
 
     private fun readFromDisk(openApiUrl: String) : String {
+
+        // file schema
         val fileScheme = "file:"
-        val path = if (openApiUrl.startsWith(fileScheme, true)) {
-            Paths.get(URI.create(openApiUrl))
-        } else {
-            Paths.get(openApiUrl)
+
+        // create paths
+        val path = try {
+            if (openApiUrl.startsWith(fileScheme, true)) {
+                Paths.get(URI.create(openApiUrl))
+            }
+            else {
+                Paths.get(openApiUrl)
+            }
+        }
+        // Exception is thrown if the path is not valid
+        catch (e: Exception) {
+            // state the exception with the error message
+            throw SutProblemException("The file path provided for the OpenAPI Schema $openApiUrl," +
+                        " ended up with the following error: " + e.message)
         }
+
+        // If the path is valid but the file does not exist, an exception is thrown
         if (!Files.exists(path)) {
-            throw SutProblemException("Cannot find OpenAPI schema at file location: $openApiUrl")
+            throw SutProblemException("The provided swagger file does not exist: $openApiUrl")
```

Comment:
use `OpenAPI` term instead of `swagger` (this latter is old name, linked to a specific company)

Reply:
Hi Andrea,

Thanks for your comment, I have replaced swagger with OpenAPI in error messages.